### ISSUE TITLE
mysql-prepare-db-dir calls obsolete option for mysql_install_db

### ISSUE THREAD

**Issue by otteydw** [iuscommunity-pkg/mysql57u#1]
The script /usr/libexec/mysql-prepare-db-dir still calls the --rpm option of /usr/bin/mysql_install_db, despite the man page saying "This option was removed in MySQL 5.7.5."

Initial mysqld service start thus gives the following error:

```
Initializing MySQL database
mysql_install_db: [ERROR] unknown option '--rpm'
2016-01-28 10:10:07 [ERROR]   Unrecognized options
Initialization of MySQL database failed.
Perhaps /etc/my.cnf is misconfigured.
```

**Comment by b-harper** [comment]
Hello @otteydw,

Thanks for taking the time to submit this issue.  Looks like [Fedora patched this issue](http://pkgs.fedoraproject.org/cgit/rpms/community-mysql.git/commit/?id=1d9080ea5c4865e4e41a88a65024c0943b657ede) and a few other things just yesterday.  We look over these changes and get updated packages into the testing repos soon.  

I would also like to draw to your attention [another issue regarding mysql57u configuration file](https://github.com/iuscommunity/wishlist/issues/35).  Seeing that you are at least evaluating mysql57u, I am curious what your thoughts on how IUS should handle configuration files.  

-Ben

**Comment by b-harper** [comment]
Updated packages have been built and will hit the testing repository after tonight's automation run.

**Comment by carlwgeorge** [comment]
This issue has been resolved since `5.7.10-2.ius` hit stable.  I confirmed that I can no longer reproduce the symptom.  Let us know if you see any other issues.
